manifests/web.pp

```diff
@@ -4,8 +4,56 @@
 #
 
 class dlrn::web(
+  $enable_https = $dlrn::enable_https,
 ){
 
+  if $enable_https {
+    $cert_file  = '/etc/pki/tls/certs/trunk_rdoproject_org.crt'
+    $cert_key   = '/etc/pki/tls/private/trunk.rdoproject.org.key'
+    $cert_chain = '/etc/pki/tls/certs/DigiCertCA.crt'
```

Comment:
I don't like the idea of hardcoding these variables. What if we set them as variables in the hiera file?